README.md

```diff
@@ -58,10 +58,18 @@ the ixbrl-viewer.
    iXBRLViewerPlugin/viewer/dist directory.
 
 [ixbrlviewer-github]: https://github.com/Workiva/ixbrl-viewer
+[CDN]: https://cdn-prod.wdesk.org/ixbrl-viewer/1.0.0/ixbrlviewer.js
```

Comment:
Are we certain this URL will work when 1.0.0 is released?  Should we release 1.0.0 and then update this README?

Reply:
Sure we can do that.

Reply:
1.0.0 is released. And the link works correctly.